smartmontools.mk

```diff
@@ -0,0 +1,45 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+STRAPPROJECTS += smartmontools 
+SMARTMONTOOLS_VERSION  := 7.2
+DEB_SMARTMONTOOLS_V    ?= $(SMARTMONTOOLS_VERSION)
+
+smartmontools-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://managedway.dl.sourceforge.net/project/smartmontools/smartmontools/7.2/smartmontools-$(SMARTMONTOOLS_VERSION).tar.gz
+	$(call EXTRACT_TAR,smartmontools-$(SMARTMONTOOLS_VERSION).tar.gz,smartmontools-$(SMARTMONTOOLS_VERSION),smartmontools)
+
+ifneq ($(wildcard $(BUILD_WORK)/smartmontools/.build_complete),)
+smartmontools:
+	@echo "Using previously built smartmontools."
+else
+smartmontools: smartmontools-setup
```

Comment:
Check the binaries created with ``otool`` for possible dependencies, then add them here as well as on your control file.